python/Ganga/new_tests/GPI/Parser/TestSelect.py

```diff
@@ -2,47 +2,68 @@
 
 from ..GangaUnitTest import GangaUnitTest
 
+job_num = 5
+job_names = ['a', 'b', 'c', 'd', 'e']
+
 class TestSelect(GangaUnitTest):
 
-    def test_SelectJob(self):
-        
-        from Ganga.GPI import Job, jobs
-        j=Job()
+    def setUp(self):
+        """Make sure that the Job object isn't destroyed between tests"""
+        super(TestSelect, self).setUp()
+        from Ganga.Utility.Config import setConfigOption
+        setConfigOption('TestingFramework', 'AutoCleanup', 'False')
```

Comment:
Ok, in that case I think we should make this more explicit. If one of the tests is checking the behaviour of a `SubJobXMLList` then it should at least say so and perhaps even do a type-check.

In general though, the jobs should be created in `setUp` and then tested in a test function. If we want to test whether `select` works on a lazy-loaded job then I guess that's another reason to look into the ability to manually 'unload' jobs from the registry.